observationtree/src/main/scala/explore/observationtree/ConstraintSetObsList.scala

```diff
@@ -407,36 +407,44 @@ object ConstraintSetObsList {
               // end of constraint set tree
               ReflexSplitter(propagate = true),
               // start of unassigned observations list
-              ReflexElement(size = 36, minSize = 36, clazz = ExploreStyles.ObsTreeSection)(
-                ReflexHandle()(
-                  Header(block = true,
-                         clazz = ExploreStyles.ObsTreeHeader |+| ExploreStyles.ObsTreeGroupHeader
-                  )(
-                    <.span(ExploreStyles.TargetLabelTitle)("UNASSIGNED OBSERVATIONS"),
-                    <.span(ExploreStyles.ObsCount, s"${unassignedObs.length} Obs")
-                  )
-                ),
-                Droppable(UnassignedObsId) { case (provided, snapshot) =>
-                  <.div(
-                    provided.innerRef,
-                    provided.droppableProps,
-                    props.getListStyle(snapshot.isDraggingOver)
-                  )(
-                    <.div(ExploreStyles.ObsTree)(
-                      <.div(ExploreStyles.ObsScrollTree) {
-                        Segment(
-                          vertical = true,
-                          clazz = ExploreStyles.ObsTreeGroup
+              ReflexElement(size = 36,
```

Comment:
Eventually we may want to persist this size, right?

Reply:
We can persist the size of each element. Reflex elements can be size aware, and there's an `onResize` callback.